compiler/src/iree/compiler/Preprocessing/Common/ConvertConvToChannelsLast.cpp

```diff
@@ -380,6 +385,11 @@ transposeConvLikeLinalgOp(PatternRewriter &rewriter, linalg::LinalgOp convOp,
     return failure();
   }
 
+  if (enableFHWCFilter)
+    // enable FHWC Filter Layout
```

Comment:
style: avoid comments that are not explaining more than what the code itself does